tests/testthat/test-download_climate.R

```diff
@@ -0,0 +1,179 @@
+# Sample regions for test
+mpios <- ColOpenData::download_geospatial("DANE_MGNCNPV_2018_MPIO")
+# Region without stations
+oicata <- mpios[which(mpios$MPIO_CDPMP == "15500"), ]
+# Regions with stations
+medellin <- mpios[which(mpios$MPIO_CDPMP == "05615"), ]
+ibague <- mpios[which(mpios$MPIO_CDPMP == "73001"), ]
+# Retrieve existing stations
+stations_test <- stations_in_roi(ibague)
+stations_names <- stations_test$codigo
+
+# Stations in ROI
+test_that("Stations in ROI throws errors", {
+  expect_error(stations_in_roi("geometry"))
+  expect_error(stations_in_roi(oicata))
+})
+
+test_that("Stations in ROI work as expected", {
+  expect_s3_class(stations_in_roi(medellin), "data.frame")
```

Comment:
Totally agree. Tests must be enhanced in all of the modules. Right now tests are only addressing inputs and general structure. This will be addressed in #64